build/configure/copy-helper.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+/**
+ * CopyHelper
+ */
+
+var utils = require('../utils');
+
+var CopyHelper = function(parentTarget, generator) {
+  this.parentTarget = parentTarget;
+  this.generator = generator;
+};
+
+CopyHelper.prototype = {
+
+  copy: function(src, dest) {
+    var srcFile = utils.getFile(src);
+
+    if (srcFile.isFile()) {
+      this.copyFile(src, dest);
+    } else if (srcFile.isDirectory()) {
+      this.copyDir(src, dest);
```

Comment:
Cool! It seems that exporting copy api would be enough for all cases, right?

I think pass srcFile to copyDir directly should be ok.